tests/test_target.py

```diff
@@ -43,7 +43,7 @@ def test_target_with_default_cpu_features():
     triple = target.Triple.host()
     target_ = target.Target(triple)
 
-@pytest.mark.skip(reason = 'CI does not have `gcc` or `clang` installed for the moment. It will be resolved once LLVM is installed.')
+@pytest.mark.skipif(platform.system() == 'Windows', reason = '`clang` is not found on the CI for the moment')
```

Comment:
`clang` is available with our custom builds. Can't we use it?

Reply:
Yup, exactly what I tried in https://github.com/wasmerio/wasmer-python/pull/467/commits/58a56759655edd60d5e8fbec44ee6873fcbd16db, but we are looking for `clang`, not `clang.exe`… We should fix that Wasmer itself, or we can symlink temporarily in our CI. Thoughts?

Reply:
> We should fix that Wasmer itself

👍 for this: can you open an issue there please?

Reply:
Yup sure